JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -1,12 +1,29 @@
-//
-//  JuiceMaker - FruitStore.swift
-//  Created by yagom. 
-//  Copyright © yagom academy. All rights reserved.
-//
-
 import Foundation
 
-// 과일 저장소 타입
+enum DefaulSetting {
+    static let stock = 10
+}
+
 class FruitStore {
+    private var fruitsInventory : [Fruit : Int] = [:]
```

Comment:
```suggestion
    private var fruitsInventory: [Fruit: Int] = [:]
```
`:` 앞에는 공백을 없애고, 뒤에만 띄워주는 게 일반적인 컨벤션입니다 🙂